### DISCUSSION TITLE
Respect HA Dark Mode?

### DISCUSSION THREAD

**Discussion by Snux** [hassio-addons/addon-adguard-home#447] [Ideas]
Hi - I didn't file this as an issue, it's more a request if it's something easy to do and that the author thinks is a good idea.  I run HA in Dark mode, easier on my eyes.  So Adguard is the only thing in the system that blinds me :)

Could something be added in General Settings to either follow the HA mode, or set some kind of Dark Mode for Adguard?

**Comment by frenck** [comment]
Add-ons have no idea. They are separate applications unrelated to Home Assistant.

../Frenck

**Reply by Snux** [reply]
Good point, I should have thought of that.  A dark mode in the Adguard settings would be nice though.  But at the bottom of the to-do-maybe list below everything else :)

**Reply by frenck** [reply]
That is up to AdGuard, add-ons wrap and make existing applications available. We don't develop AdGuard, we develop the add-on to run and install AdGuard in Home Assistant.

**Reply by Snux** [reply]
Now I feel completely stupid.  There is already a Dark Mode in general settings.  Didn't spot the picture of a Moon.
